opendebates/static/less/base.less

```diff
@@ -168,12 +168,54 @@ body > .container {
   }
 
   > .warning {
-    font-family: Knockout;
-    font-weight: bold;
     text-align: center;
-    background-color: red;
+    background-color: @brand-tertiary;
     color: white;
     padding: 10px;
+    .full({
+      min-height: 250px;
+      padding: 10px 280px;
+      position: relative;
+    });
+
+    &::before {
+      .full({
+        content: "";
+        background-image: url(/static/images/megaphone-left.png);
+        left: 15px;
+        bottom: 0;
+        height: 130px;
+        width: 176px;
+        position: absolute;
+      });
+    }
+    &::after {
+      .full({
+        content: "";
+        background-image: url(/static/images/megaphone-right.png);
+        right: 15px;
+        bottom: 0;
+        height: 130px;
+        width: 176px;
+        position: absolute;
+      });
+    }
```

Comment:
Looks like these get applied now to the Old Browser Upgrade banner that was recently added. Did you want to do that, or would you like to restrict these only to the announcement banner?